source/common/router/rds_impl.h

```diff
@@ -164,6 +169,15 @@ class RdsRouteConfigSubscription : Envoy::Config::SubscriptionCallbacks,
 
 using RdsRouteConfigSubscriptionSharedPtr = std::shared_ptr<RdsRouteConfigSubscription>;
 
+struct UpdateOnDemandCallback {
+  const std::set<std::string> aliases_;
+  std::function<void()> cb_;
+};
+
+struct ThreadLocalCallbacks : public ThreadLocal::ThreadLocalObject {
```

Comment:
rename it: The place `ThreadLocalCallbacks` is used is losing the context of on-demand